config/locales/en.yml

```diff
@@ -950,6 +951,8 @@ en:
     training_module_name: Training module
     training_module_status: Status
     up_to_date_with_training: 'up-to-date with training'
+    uploads_doc: "The total number of uploads a user has added to Wikimedia
```

Comment:
Maybe `Number of file uploads to Wikimedia Commons`.

We should either avoid the term 'course', or move this message into the `courses` section with an alternative `courses_generic` version that uses the term 'program' instead. For any message that uses terminology specific to students and courses, we always need alternative messages that don't use school-focused terms.